recipes-devtools/half/half_2.1.0.bb

```diff
@@ -17,4 +17,4 @@ do_install () {
     cp -r ${S}/include/* ${D}${includedir}
 }
 
-ALLOW_EMPTY_${PN} = "1"
+ALLOW_EMPTY:${PN} = "1"
```

Comment:
Is this correct? I'm confused about the overrides with `PN`, like `FILES`, `RDEPENDS`, `PREFERRED_PROVIDER`, and `PREFERRED_VERSION`. I guess the migration docs actually mention that these are tricky but don't really help my understanding...

Reply:
It is. `${PN}` here is an override.